e2e_tests/tests/experiment/experiment.py

```diff
@@ -511,14 +511,16 @@ def num_trials(sess: api.Session, experiment_id: int) -> int:
 
 def num_active_trials(sess: api.Session, experiment_id: int) -> int:
     return sum(
-        1
-        if t.trial.state
-        in [
-            bindings.trialv1State.RUNNING,
-            bindings.trialv1State.STARTING,
-            bindings.trialv1State.PULLING,
-        ]
-        else 0
+        (
+            1
+            if t.trial.state
+            in [
+                bindings.trialv1State.RUNNING,
+                bindings.trialv1State.STARTING,
+                bindings.trialv1State.PULLING,
+            ]
+            else 0
```

Comment:
recommend deleting `1 if` and `else 0`.  Just a nit.

Reply:
Nice. This let my remove the parens without my formatter adding them back in.